src/inmanta/ast/statements/call.py

```diff
@@ -83,8 +83,8 @@ def __init__(
 
     def normalize(self, *, lhs_attribute: Optional[AttributeAssignmentLHS] = None) -> None:
         ReferenceStatement.normalize(self)
-        self.anchors = [TypeReferenceAnchor(self.namespace, self.name)]
         func = self.namespace.get_type(self.name)
+        self.anchors = [TypeReferenceAnchor(self.namespace, self.name)]
```

Comment:
Typeanchor acted up here,so I left is as it is

Reply:
Do you have an idea why?

Reply:
yes, this is not quite a type, it might work is I move it a few lines down, but I didn't want to take risks